jkube-kit/jkube-kit-vertx/src/test/java/org/eclipse/jkube/vertx/enricher/VertxHealthCheckEnricherTest.java

```diff
@@ -540,9 +541,10 @@ void withExecTypeReadinessUsingConfig_shouldDisableLiveness() throws Exception {
 
         Probe readinessProbe = enricher.getReadinessProbe();
         assertThat(readinessProbe).isNotNull()
-            .extracting(Probe::getExec)
-            .extracting(ExecAction::getCommand).asList()
-            .hasSize(3);
+                .extracting(Probe::getExec)
+                .extracting(ExecAction::getCommand)
+                .asInstanceOf(InstanceOfAssertFactories.list(String.class))
+                .hasSize(3);
```

Comment:
could you please revert formatting changes? Have you configured your IDE to use [eclipse formatter settings](https://eclipse.dev/jkube/contributing/#intellij-idea-setup)?

Reply:
OK, I will setting my eclipse to make these formatting changes not happen,  but after my settings done, should I make a new pr or continue with this pr?

Reply:
No need to create a new PR. You can make changes and follow these steps: 
```
# After making your changes,add them to git index
git add file/you/changed
# Amend previous commit
git commit --amend
# Force push to your existing branch
git push origin master -f
```

Reply:
@Flowers2Algernon : I see you're committed changes in master branch. It's advised to create a new branch for a new change request.

Reply:
thank you for your kind advice, just now I reset my git version and now I will follow your guide to do this, thx again

Reply:
@Flowers2Algernon : No problem!